src/helper/service-utils.php

```diff
@@ -140,7 +145,11 @@ function generate_global_docker_compose_yml( Filesystem $fs ) {
 				'container_name' => GLOBAL_REDIS_CONTAINER,
 				'image'          => 'easyengine/redis:' . $img_versions['easyengine/redis'],
 				'restart'        => 'always',
-				'volumes'        => [ EE_ROOT_DIR . '/services/redis:/data' ],
+				'cmd'            => ["redis-server", "/usr/local/etc/redis/redis.conf"],
```

Comment:
There is a minor error. https://docs.docker.com/compose/compose-file/#command

```suggestion
				'command'            => ["redis-server", "/usr/local/etc/redis/redis.conf"],
```